### PR TITLE
Update RetroShare to 0.6-RC2

### PR DESCRIPTION
**Pull request by fat0troll**

### CODE REVIEW

File: Casks/retroshare.rb [hunk 1]
```diff
@@ -1,11 +1,12 @@
 cask :v1 => 'retroshare' do
-  version '0.5.5b'
-  sha256 '24beb2e1087e7a93ba996030c0785cd9f3a575d3adc5dcfda0d58aab3b40aebe'
+  version '0.6.0-RC2'
+  sha256 "94d339613f61df4a1d74fd9e7e13ca40c4773af35ffd3ab721723d58a818dfdd"
 
-  url "http://downloads.sourceforge.net/project/retroshare/RetroShare/#{version}/Retroshare-V#{version}-svn6877_OSX10.6u.dmg"
+  url "https://github.com/RetroShare/RetroShare/releases/download/v#{version}/RetroShare06-RC2.dmg.zip"
```
Comment (vitorgalvao) [inline Casks/retroshare.rb]: Needs a `# github.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (vitorgalvao) [inline Casks/retroshare.rb]: Please add a `appcast 'https://github.com/RetroShare/RetroShare/releases.atom'` line under.

File: Casks/retroshare.rb [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
   name 'RetroShare'
   homepage 'http://retroshare.sourceforge.net/'
   license :gpl
 
+  container :nested => 'RetroShare06-RC2.dmg'
```
Comment (vitorgalvao) [inline Casks/retroshare.rb]: Add empty line under this one.

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 923eb7a736879fb8072f591765d4f99e860c7b87. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**